wildfly-mixin/dashboards/wildfly-datasource.libsonnet

```
@@ -0,0 +1,533 @@
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [
+      {
+        __systemRef: 'hideSeriesFrom',
+        matcher: {
+          id: 'byNames',
+          options: {
+            mode: 'exclude',
+            names: [
+              '{__name__="wildfly_transactions_number_of_inflight_transactions", cluster="my-cluster", instance="wildfly.sample-apps.svc.cluster.local:9990", job="integrations/wildfly"}',
+            ],
+            prefix: 'All except:',
+            readOnly: true,
+          },
+        },
+        properties: [
+          {
+            id: 'custom.hideFrom',
+            value: {
+              legend: false,
+              tooltip: false,
+              viz: true,
+            },
+          },
+        ],
+      },
+    ],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local abortedTransactionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(wildfly_transactions_number_of_aborted_transactions_total{job="$job", instance="$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{instance}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Aborted transactions',
+  description: 'Number of transactions that have been aborted over time',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+
+{
+  grafanaDashboards+:: {
+    'wildfly-datasource.json':
+      dashboard.new(
+        'wildfly-datasource',
```

```suggestion
        'Wildfly datasource',
```

All links are sentence case. Here's a link on some Grafana UX principles we should abide by! 

https://grafana.com/docs/writers-toolkit/style-guide/capitalization-punctuation/#sentence-case-capitalization